cmd/keytransparency-signer/backend.go

```diff
@@ -97,10 +124,22 @@ func main() {
 		log.Fatalf("Failed to create SQL history: %v", err)
 	}
 	mutator := entry.New()
-	sths, err := appender.New(context.Background(), sqldb, *mapID, *mapLogURL, nil)
+
+	// STH Appender.
+	// The log should be in a restricted access backend environment.
+	cc, err := grpc.Dial(*logURL, grpc.WithInsecure())
```

Comment:
By the way, I really like that we can now talk to the log using gRPC instead of HTTP.